lib/engine/action/purchase_train.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Engine
```

Comment:
what is this

Reply:
A new ability on companies and the action that goes with it.

https://github.com/kelsin/18xx/blob/g_1871/lib/engine/ability/README.md#purchase_train